sormas-ui/src/main/java/de/symeda/sormas/ui/contact/importer/ContactImporter.java

```diff
@@ -356,7 +356,12 @@ private void insertColumnEntryIntoData(ContactDto contact, PersonDto person, Str
 						Pair<DistrictReferenceDto, CommunityReferenceDto> infrastructureData =
 							ImporterPersonHelper.getPersonDistrictAndCommunity(pd.getName(), person);
 						List<FacilityReferenceDto> facility = FacadeProvider.getFacilityFacade()
-							.getByName(entry, infrastructureData.getElement0(), infrastructureData.getElement1(), false);
+							.getByNameAndType(
```

Comment:
From my understanding this would be the facility of the person (occupation), so it does not make sense to use the facility type of the case. Also the case is most likely not available.

Reply:
You're right, this isn't correct. But I don't really the what would be correct here.